src/features/pinned-highlights.js

```diff
@@ -0,0 +1,35 @@
+var highlightTemplate = require('../templates/pinned-highlight');
+
+// only pin up to 10 messages
+var maximumPinCount = 10;
+
+module.exports = function(message) {
+    if (!bttv.settings.get('pinnedHighlights')) {
+        return;
+    }
+
+    var $highlightContainer = $('#bttv-pin-container');
+
+    // Push pin container to DOM if it doesn't exist
+    if (!$highlightContainer.length) {
+        $highlightContainer = $('<div id="bttv-pin-container">').appendTo($('.ember-chat .chat-room'));
+    }
+
+    // Format the time string
+    var timeSent = message.date.getHours() + ':' + ('0' + message.date.getMinutes()).slice(-2);
+
+    var $nextHighlight = $(highlightTemplate({ time: timeSent, displayName: message.tags['display-name'] + ':', message: message.message }));
```

Comment:
Why are you applying a colon here? It would make more sense if it were just a part of the template itself, no?